lecture_3/03_project/calculator/internal/domain/models/bet.go

```diff
@@ -0,0 +1,9 @@
+package models
+
+// BetCalculated represents a domain model representation of a calculated bet.
+type BetCalculated struct {
```

Comment:
`Calculated` suffix is not needed here - from the perspective of this (micro)service, those are the "bets". Of course, data model for representing those bets is not identical to the one used in Controller, but that is not relevant here. Try to think about them as completely separate services which don't know anything about each other. They only know what kind of data they consume from RabbitMQ queues and what kind of data are they supposed to produce into the queue.
Calculator service is listening to `bets` queue where bets are incoming represented as `rabbitmqmodels.Bet` and this service will map it into domain `Bet` representation (which is this file), containing only the relevant subset of information.

That's also why the table in `calc_bets` database is actually called `bets`.

The only place where you actually want to use `BetCalculated` is in `rabbitmqmodels` because you are modelling a message which will be published to RabbitMQ queue called `bets-calculated`, so `BetCalculated` is a proper naming for that.

Let me know if anything is not clear here.

Reply:
I get it now, thanks.